Class-6/script.js

```diff
@@ -0,0 +1,141 @@
+// Object Creation
+// const person={
```

Comment:
It is better to keep space before and after any operator, here we can add space before and after assignment operator,
```javascript
const person = {
```